source/server/spongineer/debugging.rst

```diff
@@ -195,28 +210,18 @@ check which plugins are involved with the crash. This requires some research as
 mod names; checking the ``Caused by`` blocks may also help.
 
 * ``java`` classes can be ignored during the error search.
-* ``net.minecraft`` is the vanilla Minecraft code. If only these elements are present, it's either a Minecraft bug or a
-  really nasty one (these are explained later).
+* ``net.minecraft`` is the vanilla Minecraft code. If only these elements are present, it's either a Minecraft bug or a 
+  coremod.
 * ``org.spongepowered`` is from Sponge itself, having only these and Minecraft packages present usually indicates a
-  Sponge bug (or a nasty one). The :doc:`bugreport` chapter explains where to report bugs.
-* Other classes have to be mapped to their mods by hand. In this case there is an entry ``com.example.extendedaiplugin`` 
-  which gives us the following hints. ``com`` probably from an English speaking mod author. This could also be a
-  language code such as ``de`` or ``zh``. The next part usually indicates the mod authors name, but in some cases it may
-  also point to a hosting platform such as ``github``. In those cases you should also take a look at the next parts.
-  After that there is usually something related to the mod itself. ``extendedaiplugin`` might be related to the plugin
-  name. Please be aware that the mod may use additional separator characters such as ``-`` or spaces, or use a different
-  character case for some parts. Beware this strategy is not always accurate, even less so in mod packs or bundles.
-
-.. note::
-
-    If you encounter a bug it is usually a good idea to create a backup first, then try to reproduce it, then narrow
-    it down by removing mods. Only then should you report the error. If the error occurs in the absence of Sponge
-    plugins, try removing SpongeForge. If the error persists its not related to Sponge. Its usually a good idea to
-    report bugs to the mod authors first as they have good knowledge of the parts of code they are working with. However
-    you can always contact us through IRC or other means. Please provide logs for bug reports, if possible.
-
-Nasty bugs: Minecraft modding uses some advanced techniques such as Mixins and ClassLoaderTransformations, which means
-that although a Minecraft class has been reported as the cause, it does not mean the code executed inside is from
-Minecraft itself. Sponge and other plugins hook into the native methods and execute their own code, such as
-posting events. In that case you have to do a blind search for the misbehaving mod. These often occur only through
-interaction between some combinations of plugins and thus are very hard to find.
\ No newline at end of file
+  Sponge bug (or another coremod being present). 
+* Other classes have to be mapped to their mods by hand. In this case there is this entry
+  ``com.example.extendedaiplugin``, Java projects usually follow the Maven package naming standard
```

Comment:
comma after ``com.example.extendedaiplugin`` should be a semicolon.
comma needed after ``groupId.groupId.artifactId``
semicolon needed after "backwards"
comma needed after ``example.com``
comma or hyphen needed after "name of the project" (or put also known as artifact in the Java world in brackets)
comma after ``extendedaiplugin` should be a semicolon
comma needed after "when in doubt"